web/atlas.json

```diff
@@ -1748,7 +1745,6 @@
 {"id": "twtmmm", "submitted_by": "DG_TBHItzDG", "name": "Kaonashi - Spirited Away", "description": "Kaonashi from the Studio Ghibli anime Spirited Away.", "website": "https://en.wikipedia.org/wiki/Spirited_Away", "subreddit": "", "center": [1419.5, 267.5], "path": [[1432.5, 278.5], [1431.5, 277.5], [1430.5, 276.5], [1429.5, 275.5], [1429.5, 273.5], [1428.5, 273.5], [1428.5, 270.5], [1428.5, 268.5], [1427.5, 267.5], [1426.5, 266.5], [1426.5, 256.5], [1425.5, 255.5], [1424.5, 254.5], [1423.5, 253.5], [1422.5, 252.5], [1422.5, 250.5], [1416.5, 250.5], [1416.5, 251.5], [1415.5, 252.5], [1414.5, 253.5], [1413.5, 254.5], [1412.5, 255.5], [1411.5, 257.5], [1411.5, 264.5], [1411.5, 269.5], [1410.5, 270.5], [1409.5, 271.5], [1409.5, 274.5], [1408.5, 275.5], [1407.5, 276.5], [1406.5, 277.5], [1405.5, 278.5], [1405.5, 279.5], [1432.5, 279.5]]},
 {"id": "twtm4y", "submitted_by": "Grace4555", "name": "Obey Me! One Master To Rule Them All", "description": "Obey me is a dating simulation app developed for android and iOS. The main character is transported to Devildom by a summoning ritual to attend Devildom Academy as a exchange student for one year. The main character will be greeted by seven gorgeous men that are avatars of the deadly sins.", "website": "https://shallwedate.jp/obeyme/en/", "subreddit": "/r/obeyme", "center": [1358.5, 394.5], "path": [[1375.5, 405.5], [1376.5, 396.5], [1376.5, 380.5], [1344.5, 382.5], [1345.5, 403.5], [1346.5, 403.5], [1345.5, 405.5], [1375.5, 405.5], [1345.5, 397.5], [1336.5, 398.5], [1337.5, 407.5], [1345.5, 407.5], [1346.5, 399.5], [1376.5, 406.5]]},
 {"id": "twtm2k", "submitted_by": "pticle94", "name": "Popcorn", "description": "Popcorn is a show on Twitch organised by the French stream PA Domingo", "website": "https://boutiquepopcorn.fr/", "subreddit": "", "center": [680.5, 1502.5], "path": [[661.5, 1521.5], [699.5, 1521.5], [699.5, 1481.5], [689.5, 1481.5], [684.5, 1486.5], [680.5, 1482.5], [662.5, 1483.5], [661.5, 1521.5]]},
-{"id": "twtm1e", "submitted_by": "Educational-Ad-170", "name": "Yuyuko", "description": "Funny ghost from Touhou project who is also a glutton", "website": "https://youtu.be/ObpEO4objqw", "subreddit": "/r/touhou", "center": [1741.5, 807.5], "path": [[1727.5, 790.5], [1728.5, 824.5], [1756.5, 824.5], [1754.5, 790.5]]},

```

Comment:
This is an individual entry for a character. It's not a duplicate of the entries for the whole artwork.